packages/dialtone-vue2/recipes/leftbar/contact_centers_row/contact_centers_row.vue

```diff
@@ -0,0 +1,213 @@
+      </div>
+      <slot name="right" />
+      <div class="dt-leftbar-row__omega">
+        <dt-badge
+          v-if="showUnreadCount"
+          data-qa="dt-leftbar-row-unread-badge"
+          kind="count"
+          type="bulletin"
+        >
+          {{ unreadCount }}
+        </dt-badge>
+        <dt-button
+          v-else
+          class="dt-leftbar-row__action"
+          data-qa="dt-leftbar-row-action-button"
+          :aria-label="menuButtonAriaLabel"
+          importance="clear"
+          size="xs"
+          circle
+          @click.stop="$emit('clickMenu', $event)"
+        >
+          <template #icon>
+            <dt-icon
+              name="chevron-down"
+              size="200"
+            />
+          </template>
+        </dt-button>
+      </div>
+    </a>
+  </div>
+</template>
+
+<script>
+import { DtBadge } from '@/components/badge';
+import { DtIcon } from '@/components/icon';
+import { DtButton } from '@/components/button';
+import DtEmojiTextWrapper from '@/components/emoji_text_wrapper/emoji_text_wrapper.vue';
+import { safeConcatStrings } from '@/common/utils';
+
+export default {
+  name: 'DtRecipeGeneralRow',
+
+  components: {
+    DtEmojiTextWrapper,
+    DtBadge,
+    DtIcon,
+    DtButton,
+  },
+
+  inheritAttrs: false,
+
+  props: {
+    /**
+     * Will be read out by a screen reader upon focus of this row. If not defined "description" will be read.
+     */
+    ariaLabel: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * Text displayed next to the icon. Required. Even if you are overriding this field using the label slot
+     * you still must input this as it will be displayed as the "title" attribute for the row.
+     */
+    description: {
+      type: String,
+      required: true,
+    },
+
+    /**
+     * Determines if the row is selected
+     */
+    selected: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * Number of unread messages
+     */
+    unreadCount: {
+      type: Number,
+      default: 0,
+    },
+  },
+
+  emits: [
+    /**
+     * Native click event on the row itself
+     *
+     * @event click
+     * @type {PointerEvent | KeyboardEvent}
+     */
+    'click',
+
+    /**
+     * Call button clicked
+     *
+     * @event call
+     * @type {PointerEvent | KeyboardEvent}
+     */
+    'clickMenu',
+  ],
+
+  data () {
+    return {
+      hovered: false,
+      labelWidth: '100%',
+    };
+  },
+
+  computed: {
+    leftbarContactCentersRowClasses () {
+      return [
+        'dt-leftbar-row',
+        'dt-leftbar-row--contact-centers',
+        {
+          'dt-leftbar-row--unread-count': this.showUnreadCount,
+          'dt-leftbar-row--selected': this.selected,
+        },
+      ];
+    },
+
+    getAriaLabel () {
+      return this.ariaLabel
+        ? this.ariaLabel
+        : safeConcatStrings([this.description, this.unreadCountTooltip]);
+    },
+
+    menuButtonAriaLabel () {
+      return 'Menu button';
+    },
```

Comment:
We don't do this on `Dialtone Vue` this needs to be a prop so the client can pass translated strings.